meta/FSMathLink.m

```diff
@@ -245,6 +249,9 @@
 CreateSpectrumDecaysInterface[modelName_] :=
     "virtual void " <> CreateSpectrumDecaysCalculationName[] <> "(const softsusy::QedQcd&, const Physical_input&, const FlexibleDecay_settings&) = 0;";
 
+CreateSpectrumUnitarityInterface[modelName_] :=
```

Comment:
I think this function is defined twice, see Line 271.

Reply:
Removed in 8e291e7.